pierone/cli.py

```diff
@@ -242,22 +210,28 @@ def tags(config, team: str, artifact, url, output, limit):
 
     if not artifact:
         artifact = get_artifacts(config.get('url'), team, token)
+        if not artifact:
+            raise click.UsageError('Either Team does not exist or currently does not artifacts in Pierone! '
+                                   'Please double check for spelling mistakes.')
+
+    registry = config.get('url')
+    if registry.startswith('https://'):
+        registry = registry[8:]
 
     slice_from = - limit
 
     rows = []
     for art in artifact:
-        r = get_tags(config.get('url'), team, art, token)
-        rows.extend([{'team': team,
-                      'artifact': art,
-                      'tag': row['name'],
-                      'created_by': row['created_by'],
-                      'created_time': parse_time(row['created']),
-                      'severity_fix_available': parse_severity(
-                          row.get('severity_fix_available'), row.get('clair_id', False)),
-                      'severity_no_fix_available': parse_severity(
-                          row.get('severity_no_fix_available'), row.get('clair_id', False))}
-                     for row in r[slice_from:]])
+        image = DockerImage(registry=registry, team=team, artifact=art, tag=None)
+        try:
+            tags = get_image_tags('pierone', image)[slice_from:]
+        except Unauthorized as e:
+            raise click.ClickException(str(e))
+        else:
+            if tags is None:
+                raise click.UsageError('Artifact or Team does not exist! '
```

Comment:
I am trying to be more user friendly and showing a msg, so people do not ask why nothing came in the output. Someone even asked me if it was a bug.